style.css

```diff
@@ -0,0 +1,812 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: #f5f8fd;
+}
+
+/* mob version */
+@media screen and (min-width: 375px) {
+  .btn {
+    display: block;
+    margin: 16px 25px 0 0;
+    cursor: pointer;
+  }
+
+  .btn-1 {
+    width: 25px;
+    height: 3px;
+    background-color: #6070ff;
+    margin-top: 3px;
+  }
+
+  .btn.active .btn-1:nth-child(2) {
+    opacity: 0;
+  }
+
+  .btn.active .btn-1:nth-child(1) {
+    background-color: white;
+    transform: translateY(6px) rotate(45deg);
+  }
+
+  .btn.active .btn-1:nth-child(3) {
+    background-color: white;
+    transform: translateY(-6px) rotate(-45deg);
+  }
+
+  .menu-sec {
+    display: flex;
+    flex-flow: column wrap;
+    position: fixed;
+    left: -600px;
+    top: 0;
+    gap: 0;
+    background-color: #6070ff;
+    opacity: 0.9;
+    height: 100%;
+    width: 100%;
+    transition: 0.3s;
+  }
+
```

Comment:
- [ ] (optional) Good job making your mobile menu design transparent  so as to match the given design. To match the Figma design even more you could create an empty `div` inside the header tag and give it a class name of overlay. Then you counld give it the following CSS properties so as to fully achieve the blurred background. You could then make the class show when the menu is clicked. You should try it.


CSS
```
.overlay {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background-color: #your color;
  backdrop-filter: blur(3px);
  z-index: 5;
}
```

HTML

```
      <header>
       <div class="overlay hidden"></div>
      </header>
```